packages/qwik-city/runtime/src/server-functions.ts

```diff
@@ -380,6 +380,18 @@ export const serverQrl = <T extends ServerFunction>(
             throw obj;
           }
           return obj;
+        } else if (contentType === 'application/json') {
+          const obj = await res.json();
+          if (res.status === 500) {
```

Comment:
Should be >=400, no?

Reply:
it should never hit this case but for example of the Lambda blows up then aws will change the return headers/value